pom.xml

```diff
@@ -1,33 +1,54 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
What is the master branch using for auto indentation? All of the whitespace for the PR were done automatically through Eclipse. My settings are spaces only with indent and tab size of 4.